introduction.md

```diff
@@ -1,2 +1,9 @@
 Hello, my name is Flavia.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?